src/components/Tables/Table/partials/Table.styled.ts

```diff
@@ -0,0 +1,21 @@
+import styled from '@emotion/styled';
+
+export const TableHead = styled.div<{ color: string }>`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: ${({ color }) => color};
+`;
+
+export const TableTitleWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+`;
+
+export const TotalRows = styled.p`
+  margin-right: 1.5rem;
+  white-space: nowrap;
+`;
+
+export const TableFooter = styled.div``;
```

Comment:
Is this still being used?

Reply:
Oh thanks, I forgot to add the styling there!